packages/clay-dropdown/src/ClayDropdownItem.soy

```diff
@@ -181,7 +184,10 @@
 			{/if}
 		"
 
-		data-onclick="{$_handleItemClick}"
+		{if not $disabled}
+			data-onchange="{$_handleItemClick}"
```

Comment:
onChange to a `li` element?

Reply:
`li` will receive the `change` event (via bubbling) when the input of the checkbox changes (being triggered by clicking on the label or directly in the input), I had to add to `li` because of the rules we have on top of it .